generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-detail.tsx

```diff
@@ -51,11 +52,11 @@ export class UserManagementDetail extends React.Component<IUserManagementDetailP
           <dt><Translate contentKey="userManagement.createdBy">Created By</Translate></dt>
           <dd>{user.createdBy}</dd>
           <dt><Translate contentKey="userManagement.createdDate">Created Date</Translate></dt>
-          <dd>{user.createdDate}</dd>
+          <dd><Time value={user.createdDate} format="DD/MM/YY hh:mm" /></dd>
```

Comment:
add the format to `app/config/constants.ts`  as `export const APP_DATE_FORMAT = 'DD/MM/YY hh:mm';` and import it and use here